src/safe_apps/serializers.py

```diff
@@ -13,7 +14,7 @@ class Meta:
 
 class ClientSerializer(serializers.Serializer[Client]):
     @staticmethod
-    def to_representation(instance: Client) -> str:
+    def to_representation(instance: Client) -> str:  # type: ignore[override]
```

Comment:
As far as I can tell, the types specify that `to_representation` _should_ return a `dict` so I set these to ignore the override.